TESTING.md

```diff
@@ -118,6 +118,11 @@ The Bottlerocket components are found in the `testsys-bottlerocket-aws` Kubernet
 
 Now that you have the testsys cluster set up, it's time to run a Bottlerocket integration test!
 
+### Configuration
+
+There are many arguments that can be configured via environment variables with `cargo make`; however, it is possible to create a configuration file instead.
+Check out the [example config file](tools/testsys/Test.toml.example) for a sample `Test.toml` file.
```

Comment:
In other places where a configuration file is used, there is documentation that explains a few fields of the configuration file, and what happens when they are set, `Test.toml` should have something similar, and you should provide snippets explaining what will happen when used.

Reply:
Do we need more documentation than the example config has?

Reply:
I won't argue about this since it seems like we already link to example configuration files to explain the different sections they have. We could consider not doing so, and do what other open source projects do: provide a document to explain the purpose of each section.